tools/ci_build/github/linux/docker/scripts/training/ortmodule/stage2/requirements.txt

```diff
@@ -1 +1,2 @@
 deepspeed
+fairscale
```

Comment:
can this be a part of stage1?

Reply:
fairscale has a dependency on torch. So will put this under stage 2